### PR TITLE
Powerbar

### PR DESCRIPTION
**Pull request by swoonzj**
Created a Power Bar. 
It generates "power" from 0-100, which changeable speed and starting power.
Currently can be tested in the Pool scene using the "Stop" button. A Text reports the current power (for debug purposes).

Created 2 new sprites:
  Moped
  Ramp 1 (a terrible ramp)

### CODE REVIEW

File: Assets/Scripts/Gameplay/PowerSlider.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    void Update () {
+
+        if (!stop)
+        {
+            power = (power + (speed * Time.deltaTime)) % 100;
+
+            transform.localScale = new Vector2(power * localYscale / 100, localYscale);
+
+            // finally updating its position to give the feeling it's growing from left to right
+            //transform.position = new Vector2(power / 2, transform.localPosition.y);
```
Comment (jmccormack200) [inline Assets/Scripts/Gameplay/PowerSlider.cs]: Remove if you're not using it.

**Review by jmccormack200 — CHANGES_REQUESTED** [review]
Look up how to rebase and squash commits.

File: Assets/Scripts/Gameplay/PowerSlider.cs (hunk 1 above)
Comment (swoonzj) [reply Assets/Scripts/Gameplay/PowerSlider.cs]: Word.

Comment (ShadowBrother) [inline Assets/Scripts/Gameplay/PowerSlider.cs]: Calling Stop() flips stop, meaning calling Stop when stop is already true, sets it to false, is this designed behavior?

File: Assets/Scripts/Gameplay/PowerSlider.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+            // For debug: output power to Text
+            GameObject.Find("txtPower").GetComponent<Text>().text = power.ToString();
+        }
+    }
+
+    public void Stop()
+    {
+        stop = !stop;
```
Comment (ShadowBrother) [inline Assets/Scripts/Gameplay/PowerSlider.cs:34]: Calling Stop() flips stop, meaning calling Stop when stop is already true, sets it to false, is this designed behavior?

Comment (swoonzj) [reply Assets/Scripts/Gameplay/PowerSlider.cs:34]: For now, yes.
That way you can keep pressing the button to start/stop the bar.

Comment (swoonzj) [reply Assets/Scripts/Gameplay/PowerSlider.cs:34]: I guess "trigger" would be a better name, but I didn't want anyone reliving any bad memories or getting angry with me.

**Pull request closed without merging**